### PR TITLE
Website v1

### PR DESCRIPTION
**Pull request by abhilashlr**

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  <img src="assets/images/central_bg.png" class="max-w-full md:max-w-xl mt-32 pt-20" alt="chennai central pic">
+</div>
+
+<div class="w-screen bg-red-dark">
+  <div class="container mx-auto p-5 text-xl">
+    <p class="text-white mb-5">
+      Hello Chennai,
+    </p>
+    <p class="text-white">
+      We are looking forward to our first Ember meet-up on August 3rd, 2019 (Saturday).
```
Comment (mirage88) [inline app/templates/index.hbs]: We look forward to

File: app/templates/index.hbs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      <div class="flex-full-center bg-grey-light p-1">
+        Gopal Venkatesan
+        <span class="text-xs ml-1">@LinkedIn</span>
+      </div>
+    </a>
+    <a class="flex flex-col sm:w-2/5 md:w-1/3 lg:w-1/5 m-2 border-4 border-red-dark rounded shadow">
+      <img src="assets/images/vivek.jpg" class="h-full w-full speaker-image object-cover" alt="">
+      <div class="flex-full-center bg-grey-light p-1">
+        Kandha Vivek Raj
+        <span class="text-xs ml-1">@ZOHO</span>
```
Comment (mirage88) [inline app/templates/index.hbs]: Switch company names with topic title

File: app/templates/location.hbs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </div>
+  </div>
+</div>
+
+<div class="container mx-auto p-10">
+  <h4>
+    Instructions:
+  </h4>
+  <ul class="text-lg mb-5">
+    <li class="py-3">Register yourself at <a href="https://www.meetup.com/Chennai-EmberJS/events/262966706/" rel="noopener" target="_blank"> EmberJS chennai meetup event. </a></li>
```
Comment (mirage88) [inline app/templates/location.hbs]: Register at

Comment (mirage88) [reply app/templates/location.hbs]: Chennai EmberJS

File: app/templates/location.hbs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <li class="py-3">For those of whom, whose seats are confirmed in the meetup event, we'll send an sms with OTP. This is to be shown at the front gate security to enter Global Infocity premises.</li>
+    <li class="py-3">Register your-self at the Freshworks front-desk.</li>
```
Comment (mirage88) [inline app/templates/location.hbs]: This order has changed. Security can be placed outside the cafeteria. Please check with Abhilash

File: app/data/schedule.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+export default [
+  {
+    startTime: "09:00 am",
+    endTime: "09:45 am",
+    name: "Registration",
+    topicDesc: "After the registration. Come to Freshworks ground floor Cafeteria."
```
[the remark this answers is not in the document]
Comment (MalayaliRobz) [reply app/data/schedule.js]: Thought registration would be at b Block.
re-writing it to: Register your-self at the desk outside Freshworks ground floor cafeteria.

File: app/data/schedule.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
+export default [
```
[the remark this answers is not in the document]
Comment (MalayaliRobz) [reply app/data/schedule.js]: pinged him the form but he did not fill yet. will update when he fills Rajsekar

added Kanjarla's data

File: app/templates/index.hbs (hunk 2 above)
Comment (MalayaliRobz) [reply app/templates/index.hbs]: ok

File: app/templates/location.hbs (hunk 4 above)
Comment (MalayaliRobz) [reply app/templates/location.hbs]: checked. its outside ground floor cafeteria

File: app/templates/index.hbs [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+<div class="flex flex-col items-center p-10">
+  <img src="assets/images/emberconf-2018.png" class="h-32" alt="tomster happy logo">
+  <div class="text-grey-dark text-2xl m-5">August 2019 Meetup featuring talks by:</div>
+
+  <div class="flex-full-center flex-wrap">
+    <a class="flex flex-col sm:w-2/5 md:w-1/3 lg:w-1/5 m-2 border-4 border-red-dark rounded shadow">
+      <img src="assets/images/gokul.jpg" class="h-full w-full speaker-image object-cover" alt="">
+      <div class="flex-full-center flex-col bg-grey-light p-1">
+        <div>Gokul Kathirvel</div>
+        <div class="text-xs ml-1">Topic: Octane. Why and Ho</div>
```
Comment (gokatz) [inline app/templates/index.hbs]: Typo: Octane. Why and Ho`w`

Comment (MalayaliRobz) [reply app/templates/index.hbs]: fixed

File: app/data/schedule.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  {
+    startTime: "09:00 am",
+    endTime: "09:45 am",
+    name: "Registration",
+    topicDesc: "Register your-self at the desk outside Freshworks ground floor cafeteria."
+  },
+  {
+    startTime: "10:00 am",
+    endTime: "10:30 am",
+    name: "Octane: Why and How",
```
Comment (abhilashlr) [inline app/data/schedule.js]: `How?`

File: app/index.html [hunk 9]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Emberjs Chennai</title>
```
Comment (abhilashlr) [inline app/index.html]: Chennai EmberJS

**Pull request merged**